common4j/build.gradle

```diff
@@ -140,12 +140,15 @@ dependencies {
 
     implementation "net.jcip:jcip-annotations:1.0"
     implementation "com.google.code.gson:gson:2.8.5"
+    implementation 'org.json:json:20180813'
 
     compileOnly "org.projectlombok:lombok:1.18.12"
     annotationProcessor "org.projectlombok:lombok:1.18.12"
 
+    testCompileOnly 'com.github.spotbugs:spotbugs-annotations:3.1.12'
```

Comment:
Why do we need to add this specifically for test? Isn't it carried over from `compileOnly`?

Reply:
nope. I actually had to do the same with Lombok.

Reply:
I guess it probably works this way in compile